### PR TITLE
Jira V2 - Fixed an issue where command reporter field in *jira-create-issue* was wrong

### PR DESCRIPTION
**Pull request by abaumgarten**
## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/30309

## Description
Fixed an issue where command reporter field in *jira-create-issue* was reporter's name instead of reporter's account ID.

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by abaumgarten** [comment]
@teizenman @guykeller i'll add to TPB a check on this field , will let you know when it is ready

**Comment by abaumgarten** [comment]
@teizenman @guykeller added TPB, now ready for CR

**Review by bakatzir — COMMENTED** [review]
Please:
1. fix the desc in the integration arg desc in the yml as well.
2. delete the issue after creating it in the TPB to not spam our instance.

**Comment by abaumgarten** [comment]
> Please:
> 
> 1. fix the desc in the integration arg desc in the yml as well.
> 2. delete the issue after creating it in the TPB to not spam our instance.

Done

File: Packs/Jira/Integrations/JiraV2/JiraV2.yml [hunk 1]
```diff
@@ -195,7 +195,7 @@ script:
     - name: assignee
       description: The name of the assignee.
     - name: reporter
-      description: The ID of the reporter.
+      description: The reporter's account Id.
```
Comment (bakatzir) [inline Packs/Jira/Integrations/JiraV2/JiraV2.yml:198]:
```suggestion
      description: The account ID of the reporter.
```

**Review by bakatzir — APPROVED** [review]

**Pull request merged**